src/components/ProductCard/ProductCard.spec.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react'
+import ProductCard from './ProductCard'
+
+describe('ProductCard', () => {
+  it('renders product card elements', () => {
+    render(<ProductCard name="productname" description="product description" />)
+
+    const image = screen.getByAltText('productname image')
+    expect(image).toBeInTheDocument()
+
+    const header = screen.getByText('productname')
```

Comment:
This should be `getByRole`
```suggestion
    const header = screen.getByRole('heading'....)
```

Reply:
done!